### PR TITLE
Create ConfirmedPlaces for existing ConfirmedTrips

### PR DESCRIPTION
**Pull request by JGreenlee**
Fixes https://github.com/e-mission/e-mission-docs/issues/865

Before confirmed_place was introduced, we created confirmed_trips without a confirmed_place. So, these existing entries are missing the confirmed_place ID for the place they should be linked to.
For those trips, we will generate a confirmed_place just-in-time and add its ID to the trip.

Once all trips have a confirmed_place, we can remove this code.

### CODE REVIEW

**Comment by shankari** [comment]
> Once all trips have a confirmed_place, we can remove this code.

And all trips should have a confirmed place after the first successful run of the pipeline that includes a `CREATE_COMPOSITE_TRIP` step

File: emission/analysis/userinput/matcher.py
Comment (shankari) [inline emission/analysis/userinput/matcher.py]: @JGreenlee how is the `confirmed_place_entry['_id')` filled in?
`Entry.createEntry` fills it in, but you are simply using `new Entry()`
I assumed that you would get the id back from the insert call and use it

Can you verify that the `_id` field is actually non-empty?

```suggestion
            confirmed_place_entry = create_confirmed_place_entry(ts, cleaned_place)
            cpeid = ts.insert(confirmed_place_entry)
            ct["data"]["confirmed_place"] = cpeid
            logging.debug("Setting the confirmed_place key to the newly created id %s" % cpeid)
```

**Review by shankari — CHANGES_REQUESTED** [review]
@JGreenlee Looks good overall - just want to confirm that one part is actually working before merging
Maybe confirm from the logs that the id is actually getting filled in correctly

File: emission/analysis/userinput/matcher.py
Comment (JGreenlee) [reply emission/analysis/userinput/matcher.py]: `_id` was empty before being added to the timeseries. That's how the first commit was.

The second commit calls ts.insert() first, after which `_id` does exist.

Comment (JGreenlee) [reply emission/analysis/userinput/matcher.py]: The suggestion looks cleaner though. Let's use that.

**Review by shankari — APPROVED** [review]
LGTM! Let's deploy this and see if it works

**Pull request merged**
